File: easy/399/main.go
package main

import "fmt"

func lettersum(s string) int {
	t := 0
	for _, c := range []byte(s) {
		t += int(c - 96)
	}
	return t
}

func main() {
	fmt.Println(`lettersum("") => `, lettersum(""))
	fmt.Println(`lettersum("a") => `, lettersum("a"))
	fmt.Println(`lettersum("z") => `, lettersum("z"))
	fmt.Println(`lettersum("cab") => `, lettersum("cab"))
	fmt.Println(`lettersum("excellent") => `, lettersum("excellent"))
	fmt.Println(`lettersum("microspectrophotometries") => `, lettersum("microspectrophotometries"))
}

File: easy/399/main.py
def lettersum(s):
    return sum(ord(c) - 96 for c in s)


assert lettersum("") == 0
assert lettersum("a") == 1
assert lettersum("z") == 26
assert lettersum("cab") == 6
assert lettersum("excellent") == 100
assert lettersum("microspectrophotometries") == 317
